tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -1886,62 +1886,64 @@ STATIC_ASSERT(ranges::viewable_range<std::span<int>>);
 STATIC_ASSERT(ranges::viewable_range<std::span<int> const>);
 
 namespace poison_pill_test {
+    static int some_int = 42;
+
     template <class T>
-    auto begin(T&) {
-        STATIC_ASSERT(always_false<T>);
+    int* begin(T&) {
+        return &some_int;
     }
     template <class T>
-    auto begin(const T&) {
-        STATIC_ASSERT(always_false<T>);
+    int const* begin(T const&) {
+        return &some_int;
     }
     template <class T>
-    auto end(T&) {
-        STATIC_ASSERT(always_false<T>);
+    int* end(T&) {
+        return &some_int + 1;
     }
     template <class T>
-    auto end(const T&) {
-        STATIC_ASSERT(always_false<T>);
+    int const* end(T const&) {
+        return &some_int + 1;
     }
     template <class T>
     auto rbegin(T&) {
-        STATIC_ASSERT(always_false<T>);
+        return std::reverse_iterator<int*>{&some_int + 1};
     }
     template <class T>
-    auto rbegin(const T&) {
-        STATIC_ASSERT(always_false<T>);
+    auto rbegin(T const&) {
+        return std::reverse_iterator<int*>{&some_int + 1};

```

Comment:
This says `<int*>`, but it seems that it should say `<int const*>`. Occurs below.